test/system/030-run.bats

```diff
@@ -29,6 +29,16 @@ echo $rand        |   0 | $rand
         run_podman $expected_rc run $IMAGE "$@"
         is "$output" "$expected_output" "podman run $cmd - output"
     done < <(parse_table "$tests")
+
+    check_no_unprivileged_access
+}
+
+@test "podman run - basic tests --uidmapping"  {
+    run_podman --uidmapping 0:10000:10000 run $IMAGE true
+
+    run_podman --uidmapping 0:10000:10000 -v foo:/foo run $IMAGE true
```

Comment:
Neither of these work. Hint: you can test via `PODMAN=/path/to/podman bats test/system/030-run.bats`

There are other problems; I'm working on some modifications to the framework, will submit them as a PR on your PR.